odoo/addons/base/models/res_partner.py

```diff
@@ -112,8 +112,9 @@ class PartnerCategory(models.Model):
     def _get_default_color(self):
         return randint(1, 11)
 
-    name = fields.Char(string='Tag Name', required=True, translate=True)
-    color = fields.Integer(string='Color', default=_get_default_color)
+    name = fields.Char(string='Name', required=True, translate=True)
+    display_name = fields.Char(string='Display Name', compute='_compute_display_name', search='_search_display_name')
```

Comment:
You can omit string and compute as they're already defined on the default field in `models.py`